src/client/rooms/room-page.js

```diff
@@ -35,27 +33,50 @@ export class RoomPage extends Component {
                 <h1>{`Room: ${this.props.roomData.name}`}</h1>
                 <UserList users={this.props.roomUsers} />
                 <div className="col-lg-8" >
-                    <button id="start-game" className="btn btn-success">Start Game</button>
+                    <button id="start-game" className="btn btn-success" disabled={disable.bind(this)()} 
+                            onClick={() => {messageStartGame()}}>Start Game</button>
                     <button id="leave-room" className="btn btn-danger" onClick={leaveRoom}>Leave Room</button>
                 </div>
+                {this.props.start ? <Game/> : null}
             </div>
         );
     }
 };
 
+function disable() {
+    if(enoughPlayers.bind(this)()){
+        return !isCreator();
+    }
+    return true;
+}
+function enoughPlayers() {
+    return this.props.roomUsers.length >= 2 ? true : false;
+}
+
+function isCreator() {
+    var room = getRoom();
+    var creator = room.data.creator;
+    var user = getUser();
+    return JSON.stringify(creator) === JSON.stringify(user);
+}
+
 function leaveRoom() {
     router.navigateToLobby();
 }
 
 const mapStateToProps = (state, ownProps) => ({
     roomUsers: state.roomUsers,
-    roomData: state.roomData
+    roomData: state.roomData,
+    start: state.startGame
 });
 
 const mapDispatchToProps = dispatch => {
```

Comment:
There's a neat trick when all you're doing is returning an object - grab me after lunch